### DISCUSSION TITLE
Is there Hyperparameter project for graphConv same as SAGEConv

### DISCUSSION THREAD

**Discussion by harirRzi** [pyg-team/pytorch_geometric#9304] [Q&A]
Hello. I used a SAGEConv layer with hyper parameter project=True for my model and the prediction is very well. Now I want to improve the prediction of it by using GraphConv using aggr='mean' and edge_weight that predicts similar to SAGEConv without project=True. I checked PyTorch website and I didn't see any hyper parameter 'project' for GraphConv. Is there any alternative for this hyper parameter?

**Comment by rusty1s** [comment]
`project` is specific to `SAGEConv`. You can achieve a similar thing via
```
self.conv(self.lin(x), edge_index)
```
but IMO it is not a crucial parameter.
